app/models/tslr_claim.rb

```diff
@@ -20,6 +21,8 @@ class TslrClaim < ApplicationRecord
     "2019-2020",
   ].freeze
 
+  TRN_LENGTH = 7
```

Comment:
We should explicitly `freeze` this constant to make it immutable.

Reply:
Phew, that's what we thought!